owncloudApp/src/main/java/com/owncloud/android/workers/UploadFileFromContentUriWorker.kt

```diff
@@ -60,7 +61,8 @@ import java.io.File
 
 class UploadFileFromContentUriWorker(
     private val appContext: Context,
-    private val workerParameters: WorkerParameters
+    private val workerParameters: WorkerParameters,
+    private val connectionValidator: ConnectionValidator
```

Comment:
This approach won't work. We do not inject anything via parameters in the workers. We need to find another way to do it.

1. We could inject it later as we do sometimes in other workers, check [camera uploads worker](https://github.com/owncloud/android/blob/master/owncloudApp/src/main/java/com/owncloud/android/workers/CameraUploadsWorker.kt#L71)
2. If we need to inject it, we could follow [koin guidelines for workmanager](https://insert-koin.io/docs/reference/koin-android/workmanager/)
3. We could try to inject the connectionValidator directly in the ClientManager if possible

Reply:
A rollback to the [previous approach](https://github.com/owncloud/android/pull/3456/files/add5b5dd399540df81fe1a87e6fcd59b60af5188#diff-5332e04954815da4d22dc0c04385920f2b1e357b1e8d6c29758ca769bd17c833R234) should work too